usaspending_api/common/tests/test_text_helpers.py

```diff
@@ -0,0 +1,51 @@
+from usaspending_api.common.helpers.text_helpers import slugify_text_for_file_names
+
+
+def test_slugify_text_for_file_names():
+    assert slugify_text_for_file_names("test") == "test"
+    assert slugify_text_for_file_names("test", "default") == "test"
+    assert slugify_text_for_file_names("test", "default", 50) == "test"
+    assert slugify_text_for_file_names("test", "default", 2) == "te"
+
+    assert slugify_text_for_file_names(None) is None
+    assert slugify_text_for_file_names(None, None) is None
+    assert slugify_text_for_file_names(None, "default") == "default"
+    assert slugify_text_for_file_names(None, "default", 50) == "default"
+    assert slugify_text_for_file_names(None, "default", 2) == "default"
```

Comment:
Seems like either option of:
- returning the full default string when it exceeds the "max_length" of characters, or
- a substring of the default string with length "max_length"

could be counter-intuitive. Any ideas?

Reply:
Not sure why returning the default string if it exceeds max_length is counter-intuitive.  I have to trust that the caller knows what they're doing.  If they give me a default, I'm going to want to return the default.  If they need the default to follow the same rules, they're more than welcome to call this function on the default value as well.

    slugify_text_for_file_names(
        'my string',
        slugify_text_for_file_names('my default')
    )

Reply:
Good point on the sentinel value